vignettes/invariants.md

```diff
@@ -952,10 +953,12 @@ Exception: OOB values generate warnings instead of errors:
 <td>
     tbl[10, ]
 
-    #> Warning: Row indexes must be between 0
-    #> and the number of rows (4). Use `NA` as
-    #> row index to obtain a row full of `NA`
-    #> values.
+    #> Warning: The `i` argument of
+    #> ``[.tbl_df`()` if integer must be
```

Comment:
Too many `` ` ``s

Reply:
Added by {lifecycle}. What should this look like?